src/application/clients/crux-wallet-client.ts

```diff
@@ -433,9 +442,16 @@ export class CruxWalletClient {
             throw ErrorHelper.getPackageError(null, PackageErrorCode.CouldNotFindBlockstackConfigurationServiceClientConfig);
         }
         this.cruxAssetTranslator = new CruxAssetTranslator(this.cruxDomain.config.assetMapping, this.cruxDomain.config.assetList);
+        if (options.disableCruxMessenger) {
+            return;
+        }
         const selfIdClaim = await this.getSelfClaim();
         if (selfIdClaim) {
-            await this.setupCruxMessenger(selfIdClaim);
+            try {
+                await this.setupCruxMessenger(selfIdClaim, options);
+            } catch (err) {
+                console.log(err);
```

Comment:
What is the purpose of this? Remove if it was for your personal debugging only.

Reply:
actually it was for catching errors from setupCruxMessenger but now we can remove it(options are being used till here only)

Reply:
'catching error' is not a valid purpose. 'catching error to _________' is a valid purpose

1. Catching error to silence/suppress it 
2. Catching error to transform it and throw it again as another error
3. Catching error to retry
4. etc


Here you've suppressed it and logged it. Any particular cases/errors you're trying to catch here? If this function is too failure prone then it may block other functionalities of SDK. So then it may be a good idea to suppress error here like youve done
But in that case we should figure out why its failing and fix that.